src/test/java/com/ss/ita/econews/ui/econews/CommentTest.java

```diff
@@ -71,8 +76,19 @@ public void verifyCommentDeletingWithRepliesTest() {
 
     @Test
     public void verifyReviewRepliesTest() {
-        driver.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS);
-
+        String replyText = "Reply";
+        NewsListCommentComponent newsListCommentComponent = new NewsPage(driver)
+                .createAndPublicComment("Comment")
+                .getCommentByIndex(0)
+                .createOneReplyToComment(replyText);
+        for (int i = 0; i < 10; i++) {
+            newsListCommentComponent.createAnotherReply(replyText);
+        }
+        newsListCommentComponent.clickViewReplyButton();
+        int numberOfRepliesOnFirstSection = newsListCommentComponent.numberOfReplies();
+        newsListCommentComponent.clickNextRepliesButton();
+        int numberOfRepliesOnSecondSection = newsListCommentComponent.numberOfReplies();
+        assertTrue(numberOfRepliesOnFirstSection + numberOfRepliesOnSecondSection == 11);
```

Comment:
int replayCount = 10

for (int i = 0; i < replayCount ; i++) 
...

  assertTrue(numberOfRepliesOnFirstSection + numberOfRepliesOnSecondSection == replayCount +1);